src/SFA.DAS.Tools.Support.Web/Views/Approvals/StopApprenticeshipSubmitted.cshtml

```diff
@@ -0,0 +1,70 @@
+﻿@{
+    ViewData["Title"] = "Support Portal";
+    @model StopApprenticeshipViewModel
```

Comment:
Since this viewmodel is used for editing and has validation logic embedded, suggest use a different viewmodel here as this is readonly view.